README.md

```diff
@@ -55,6 +55,13 @@ Relative path to file with additional documentation in markdown.
 - `attachments`
 Dictionary of text keys and URI values to additional, relevant files.
 
+- [`pacakged_by`]
+The person(s) that have packaged and uploaded this model. Only needs to be specified if different from `authors` in the root weights`, see `weights` for more details.
+
+- [`parent`] Parent model from which this model has been derived, e.g. by finetuning the weights of this model on a different dataset.
```

Comment:
As there is `parent` here and in weights, could it be more specific? For example:

```yaml
- [`parent`] Parent model from which the trained weights of this model have been derived, e.g. by finetuning the weights of this model on a different dataset. For format changes of the same trained model checkpoint, see `models`.
```

Reply:
Thanks, I added your suggestion.